about.html

```diff
@@ -7,67 +7,74 @@
-        
-      <footer>
-        <nav>
-          <a href="index.html">Home</a>
-          <a href="#">About</a>
-          <a href="#">Products</a>
-          <a href="#">Blog</a>
-          <a href="#">Contact</a>
-        </nav>
-      </footer>
-    </div><!-- container -->        
+	<nav>
+		<header>
+			<header class="topLinks">
+				<img src="/img/lambda-black.png">
+				<a href="/index.html"> Home</a>
+				<a href="#"> About </a>
+				<a href="#"> Products</a>
+				<a href="#"> Blog</a>
+				<a href="#"> Contact</a>
+			</header>
+		</header>
+	</nav>
+	<div>
+		<div class="top-img">	
+			<img class="about-header-img" src="/img/jumbo-about.png" >
+		</div>		
+		<div>		
+			<section>
+					<h1>About</h1>
+						<p>agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+			</section>
+		</div>	
+	</div>
+	<div>			
+		<div class="flexbox">
+			<div id="strategy">
+				<img src="img/about-plan.png" alt="strategy">
+				<h3>Strategy</h3>
+					<article>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</article>
+						<button> Learn More</button>
+			</div>
+			<div id="how">
+				<img src="img/about-working.png" alt="strategy">
+				<h3> How We Work </h3>
+					<article>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</article>
+						<button> Learn More </button>
+			</div>
+			<div id="places">
+				<img src="img/about-office.png" alt="strategy">
+				<h3>Places We Work</h3>
+					<article>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. </article>
+						<button>Learn More</button>
+			</div>		
+			<div id="Collab">				
+				<img src="img/about-meeting.png" alt="strategy">
+				<h3>Collaboration </h3>
+					<article>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</article>
+						<button> Learn More </button>
+			</div>
+		</div>
+	</div>
+	<footer>
+<div class="bottom">
+<h3> Lets Work Together </h3>
+<article>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</article>
+</div>
+<footer id="foot">
+	<a href="index.html">Home</a>
+	<a href="#">About</a>
+	<a href="#">Products</a>
+	<a href="#">Blog</a>
+	<a href="#">Contact</a>
+</footer>
 </body>
+
```

Comment:
The bulk of this page should be wrapped in a container. Like a div or a main tag, or something.

That container should be 100% wide, max 800px, and centered.